test/services/return-requirements/submit-points.service.test.js

```diff
@@ -0,0 +1,185 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Things we need to stub
+const FetchPointsService = require('../../../app/services/return-requirements/fetch-points.service.js')
+const PointsValidation = require('../../../app/validators/return-requirements/point.validator.js')
+
+// Thing under test
+const SubmitPointsService = require('../../../app/services/return-requirements/submit-points.service.js')
+
+describe('Submit Points service', () => {
+  let payload
+  let session
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+
+    session = await SessionHelper.add({
+      data: {
+        licence: {
+          id: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
+          currentVersionStartDate: '2023-01-01T00:00:00.000Z',
+          endDate: null,
+          licenceRef: '01/ABC',
+          licenceHolder: 'Turbo Kid',
+          startDate: '2022-04-01T00:00:00.000Z'
+        },
+        journey: 'returns-required'
+      }
+    })
+  })
+
+  afterEach(() => {
+    Sinon.restore()
+  })
+
+  describe('when called', () => {
+    describe('with a valid payload', () => {
+      beforeEach(() => {
+        payload = {
+          points: [
+            'At National Grid Reference TQ 69212 50394 (RIVER MEDWAY AT YALDING INTAKE)'
+          ]
+        }
+
+        Sinon.stub(FetchPointsService, 'go').resolves([
+          {
+            NGR1_EAST: '69212',
+            NGR2_EAST: 'null',
+            NGR3_EAST: 'null',
+            NGR4_EAST: 'null',
+            LOCAL_NAME: 'RIVER MEDWAY AT YALDING INTAKE',
+            NGR1_NORTH: '50394',
+            NGR1_SHEET: 'TQ',
+            NGR2_NORTH: 'null',
+            NGR2_SHEET: 'null',
+            NGR3_NORTH: 'null',
+            NGR3_SHEET: 'null',
+            NGR4_NORTH: 'null',
+            NGR4_SHEET: 'null'
+          },
+          {
+            NGR1_EAST: '68083',
+            NGR2_EAST: 'null',
+            NGR3_EAST: 'null',
+            NGR4_EAST: 'null',
+            LOCAL_NAME: 'BEWL WATER RESERVOIR',
+            NGR1_NORTH: '33604',
+            NGR1_SHEET: 'TQ',
+            NGR2_NORTH: 'null',
+            NGR2_SHEET: 'null',
+            NGR3_NORTH: 'null',
+            NGR3_SHEET: 'null',
+            NGR4_NORTH: 'null',
+            NGR4_SHEET: 'null'
+          }
```

Comment:
It looks like this data is used in a couple of places. And because of its size it tends to push the tests down when that is what we're trying to work with.

If you scout our code base in cases like this, you'll find we often put a function at the end of the tests to return test data. That way we remove the duplication and convert things like setting the resolver in a stub to be a one-liner.

I'd suggest doing that here.